subsys/bluetooth/host/conn.c

```diff
@@ -1456,6 +1456,11 @@ void bt_conn_set_state(struct bt_conn *conn, bt_conn_state_t state)
 
 		bt_l2cap_connected(conn);
 		notify_connected(conn);
+		/*
+		 * Inform the connection handle about the connection
+		 * parameters
+		 */
+		notify_le_param_updated(conn);
```

Comment:
Connected callback above is sufficient, new function call adds to added CPU usage.